tests/unit/beacons/test_log.py

```diff
@@ -6,7 +6,7 @@
 import logging
 
 # Salt libs
-import salt.beacons.log_beacon as log_beacon
+import salt.beacons.log as log_beacon
```

Comment:
I think this would be part of that. Why did we rename `salt/beacons/log_beacon.py` to `log.py`?

FWIW, it's failing because https://github.com/saltstack/salt/blob/master/salt/beacons/__init__.py#L19 is defined so this attempt at import doesn't work.

Reply:
Rename was done to match the name to what the tests expected. Great catch. Will revert and update the tests instead.

Reply:
Probably the general pattern to follow - I think there were  a couple of other file renames, too.

Reply:
log beacon changes have been reverted, and tests pass now